See Mozilla bug #675585 - crash [@ g_slice_free1] when run firefox-bin -no-
remote -process-updates.

diff --git a/xpcom/glue/standalone/nsXPCOMGlue.cpp b/xpcom/glue/standalone/nsXPCOMGlue.cpp
--- a/xpcom/glue/standalone/nsXPCOMGlue.cpp
+++ b/xpcom/glue/standalone/nsXPCOMGlue.cpp
@@ -386,6 +386,8 @@
     }   
   }
 
+  static void* gthreadso = dlopen("libgthread-2.0.so.0", RTLD_GLOBAL | RTLD_LAZY);
+
   GetFrozenFunctionsFunc sym =
     (GetFrozenFunctionsFunc)GetSymbol(sTop->libHandle,
                                       LEADING_UNDERSCORE "NS_GetFrozenFunctions");
